Automattic/woocommerce-payments
# Reporting: Track events for `View report` links on Payment activity widget

Fixes Automattic/woocommerce-payments#8686

#### Changes proposed in this Pull Request

Add track events to track clicks on `View report` links on the following tiles:
* Total payment volume
* Charges
* Refunds
* Disputes

#### Testing instructions

* Enable feature flag, wp option update _wcpay_feature_payment_overview_widget 1.
* Use [Tracks Vigilante](https://github.com/Automattic/tracks-vigilante-2-chrome-extension) chrome extension to track clicks from your local test site
* Check out this branch and run `npm start` or `npm run watch` to build assets
* Browse to `Payments` and on the overview page, click on the `View report` link that you see over `Total payment volume` tile. Click the link
* Check the tracked events on the Tracks Vigilante chrome extension. You should see the event `wcpay_overview_total_payment_volume_view_report_clicks` . Check the other details shared with the event, including the `source` that should read as `total_payment_volume_tile_view_report`
* Test the links on `Charges`, `Refunds` and `Disputes` tile too and see if they generate relevant track events.
###### Reminder: test this change will achieve the desired impact/result!
Reviewers please keep in mind the flow and funnel that we want to measure, and check that this event will allow us to measure the desired funnel conversion rate. i.e. test the whole flow, not just that this event fires.

- Can measure conversion rate of: View payments overview ⇒ click `View report` ⇒ view relevant report screen

See the project thread on P2 for more details and example funnel. 


#### Notes

Discussion on naming events - p1713786808704209-slack-C06BBFTF6EM

-------------------


**Post merge**
